server/src/main/java/org/spine3/server/storage/memory/InMemoryStandStorage.java

```diff
@@ -46,6 +46,8 @@
 public class InMemoryStandStorage extends StandStorage {
 
     private final InMemoryRecordStorage<AggregateStateId> recordStorage;
+    private static final String TYPE_URL_MISMATCH_MESSAGE_PATTERN
```

Comment:
Could you please declare the constant before any instance fields? Usually `static`fields are declared first.